core/src/components/button/test/form-reference/button.e2e.ts

```diff
@@ -0,0 +1,107 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ directions: ['ltr'], modes: ['ios'] }).forEach(({ title, config }) => {
```

Comment:
This used to test all configs, but it's not needed so I have it only check LTR + iOS mode since it's just checking behavior